files/app/main/status/e/backup-config.ut

```diff
@@ -0,0 +1,51 @@
+{%
+/*
+ * Part of AREDN® -- Used for creating Amateur Radio Emergency Data Networks
+ * Copyright (C) 2024 Tim Wilkinson
+ * See Contributors file for additional contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Additional Terms:
+ *
+ * Additional use restrictions exist on the AREDN® trademark and logo.
+ * See AREDNLicense.txt for more info.
+ *
+ * Attributions to the AREDN® Project must be retained in the source code.
+ * If importing this code into a new or existing project attribution
+ * to the AREDN® project must be added to the source code.
+ *
+ * You must not misrepresent the origin of the material contained within.
+ *
+ * Modified versions must be modified to attribute to the original source
+ * and be marked in reasonable ways as differentiate it from the original
+ * version
+ */
+%}
+{%
+if (request.env.REQUEST_METHOD === "PUT") {
```

Comment:
I presume this method/page is only for downloading the backup file? If so, the only acceptable verb should be `GET`.

Also, strictly speaking, for a REST API, this would return a 405, with a body containing details on the acceptable verbs. Does the UI send a header or other token that could be used to distinguish?

```suggestion
if (request.env.REQUEST_METHOD !== "GET") {
```

Reply:
It's not a REST API.
Yes this page is used to download the backup file, but it's not accessible via GET because we dont protect GETs from guest access.  Could I do that? - sure - but it's easier just to put something requiring admin access behind a PUT. This triggers a browser download and I'm not entirely sure what it'd do with a 405. But there's no application in the browser to interpret it anyway.